region/crateria/central.json

```diff
@@ -1853,17 +1862,21 @@
             {
               "name": "Pit Room Right Grey Lock (to Elevator)",
               "lockType": "killEnemies",
-              "lock": [ "f_ZebesAwake" ],
+              "lock": [
+                    "Missile",
+                    "Morph"
+                  ],
               "unlockStrats": [
                 {
                   "name": "Base",
                   "notable": false,
                   "requires": [],
-                  "note": "The enemies can be killed with just Power Beam."
+                  "note": "This door is unlocked until having morph and missiles, and then free to open after with just power beam."
                 }
               ]
             }
-          ]
+          ],
+          "yields": [ "f_ZebesAwake" ]
```

Comment:
I think we have a modeling problem here. Currently, the flag from a `yields` is obtained by interacting with a node. By chance, the left door works, but it means the right door is modeled as yielding the flag even if the lock itself isn't active.

Reply:
Resolved by #487